### PR TITLE
Fix small golint warning

### PR DESCRIPTION
**Pull request by MaXinjian**
Fix golint warning: if block ends with a return statement, so drop this else and outdent its block

ALLOW_MODULE_SPAN

### CODE REVIEW

File: functions/examples/injection-tshirt-sizes/image/main.go [hunk 1]
```diff
@@ -68,13 +68,12 @@ func size(r *yaml.RNode) error {
...
-	} else {
-		// lookup the memory and cpu quantities based on the tshirt size
-		memorySize = memorySizes[size]
-		cpuSize = cpuSizes[size]
-		if memorySize == "" || cpuSize == "" {
-			return fmt.Errorf("unsupported tshirt-size: " + size)
-		}
+	}
+	// lookup the memory and cpu quantities based on the tshirt size
+	memorySize = memorySizes[size]
```
Comment (KnVerey) [inline functions/examples/injection-tshirt-sizes/image/main.go:73]: The `size` variable is set by the `if`, so this code is invalid after this change (`size` is now referring to this function, whose name the variable shadows)

**Comment by natasha41575** [comment]
Please squash this change into 1 commit

**Comment by MaXinjian** [comment]
> Please squash this change into 1 commit

Done

**Pull request merged**